### PR TITLE
add script for symlinking paths in $EPREFIX to corresponding paths in host root

### PR DESCRIPTION
**Pull request by boegel**
cfr. #15

### CODE REVIEW

File: prefix-symlink-host-paths.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+# /etc/passwd: required to ensure local users are known (see https://github.com/EESSI/compatibility-layer/issues/15)
+# /etc/group: required to ensure local user groups are known
+for path in /etc/passwd /etc/group; do
+    echo ">> checking $path ..."
+    ls -ld ${EPREFIX}$path | grep " -> $path" > /dev/null
+    ec=$?
+    if [ $ec -ne 0 ]; then
+        echo_yellow ">> [CHANGE] ${EPREFIX}$path is *not* a symlink to $path, fixing that..."
+        rm ${EPREFIX}$path
```
Comment (bedroge) [inline prefix-symlink-host-paths.sh:34]: If we want to do the same for directories (as CC suggested), this will have to be a bit more aggressive ;-)

**Review by bedroge — COMMENTED** [review]
Looks okay to me, though I was working on something similar with Ansible and I would prefer to have it there: I think that's cleaner, and it keeps all the configuration for the compatibility layer in a single file.

File: prefix-symlink-host-paths.sh (hunk 1 above)
Comment (boegel) [reply prefix-symlink-host-paths.sh:34]: Sure, but since it's only files right now, I'd keep it at that for now.

**Comment by boegel** [comment]
> Looks okay to me, though I was working on something similar with Ansible and I would prefer to have it there: I think that's cleaner, and it keeps all the configuration for the compatibility layer in a single file.

That's not entirely true, since the set of packages to install is already in a separate file too...

There's value in being able to run the steps "manually" as well outside of Ansible imho?

**Comment by bedroge** [comment]
True, but that file had to be part of the overlay repo. If I would use the Ansible playbook for installing the compatibility layer, I would prefer to have it all together (as much as possible), instead of having to edit both my variables file and bash scripts. I do see the added value of being able to run things manually, though. And since we already describe a manual procedure in the README anyway, maybe we should split it all up and make two different dirs with Ansible and bash stuff?

**Comment by boegel** [comment]
@bedroge Splitting things up makes sense to me...

**Review by bedroge — CHANGES_REQUESTED** [review]
Could you also update .github/workflows/ansible-lint.yml and change line 26:
```
        targets: "playbooks/*.yml"
```
to:
```
        targets: "ansible/playbooks/*.yml"
```

**Comment by boegel** [comment]
@bedroge Nice catch... Done in 0a5f04b

**Review by bedroge — APPROVED** [review]
lgtm

**Pull request merged**